zulip/zulip/__init__.py

```diff
@@ -1076,6 +1076,21 @@ def delete_stream(self, stream_id):
             method='DELETE',
         )
 
+    def get_member(self, user_id, request=None):
+        # type: (int, Optional[Dict[str, Any]]) -> Dict[str, Any]
+
+        '''
+            Example usage:
+
+            >>> client.get_member(8, {'include_custom_profile_fields': True})
+            {'result': 'success', 'msg': '', 'user': [{...}, {...}]}
+        '''
+        return self.call_endpoint(
+            url='users/'+str(user_id),
```

Comment:
We generally do space around operators like `+`; see PEP-8 which we follow on this.

But probably it's better to do string formatting, rather than `+`, here.